### ISSUE TITLE
Vagrant box install issue, "failed to create the raw output file"

### ISSUE THREAD

**Issue by lhenze** [monicahq/monica#1203]
error: 

```
Stderr: VBoxManage: error: RawFile#0 failed to create the raw output file /home/asbin/devel/monica/monica3/scripts/vagrant/build/ubuntu-xenial-16.04-cloudimg-console.log (VERR_FILE_NOT_FOUND)
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component ConsoleWrap, interface IConsole
```

I've tried two different Mac Operating Systems (High Sierra and El Capitan) , and two different Virtualbox versions (5.2.10 and 5.1.26).  On High Sierra, In either Virtualbox version, I can see the monica box set up up next to another vagrant box I have running, but the other vagrant box works well.  
For Monica HQ, I consistently get this error, any ideas?

**Comment by djaiss** [comment]
@asbiin do you have any ideas of what might happen here?

**Comment by asbiin** [comment]
Well, the reason is I've fuck'd up somewhere !
I dont understand yet where is the glitch, I will fix this these week.

**Comment by asbiin** [comment]
@lhenze can you try version 2.0.1 please ?

**Comment by MaMuk** [comment]
Hi I tried it with the Version 2.1.0 which I got when I ran vagrant box update. But I still get this Error.
I found the offending Setting in the file:
~/.vagrant.d/boxes/monicahq-VAGRANTSLASH-monicahq/2.1.0/virtualbox/box.ovf
```
<UART>
    <Port slot="0" enabled="true" IOBase="0x3f8" IRQ="4" path="/home/asbin/devel/monica/monica-v2.1.0/scripts/vagrant/build/ubuntu-xenial-16.04-cloudimg-console.log" hostMode="RawFile"/>
</UART>
```
I don't know what exactly that setting is doing, or if I'm breaking something when removing it, but doing so fixed the issue.
